app/styles/main.css

```diff
@@ -783,6 +784,7 @@ section{word-break:break-word;}
   h1.hero-head{font-size:1.65rem;}
   .hero{padding-top:3%;padding-bottom:75px;}
   .hero-lead{line-height:1.125rem;}
+  .no-mobile-paddings{padding-left:0px; padding-right: 0px;}
```

Comment:
I actually need your opinion on this, it is a detail. 

Bootstrap `container` introduces `15px` left and right paddings. I feel keeping these paddings produces the required result for wide screens because the video is perfectly aligned to the paragraphs below. On the other hand, this `15px paddings` seem a little weird on mobile (and maybe tablet) screens. This is why I felt we need different paddings depending on screen width. I will post some screenshots below. Let me know about your preference on this and I will adapt the implementation.

![wide-screen-with-paddings](https://user-images.githubusercontent.com/4354335/29658635-4fd1747a-88c4-11e7-83c6-b91c4c519171.png)

![wide-screen-no-paddings](https://user-images.githubusercontent.com/4354335/29658651-5de8568c-88c4-11e7-84ce-1c07c532a1d0.png)

![mobile-screen-paddings](https://user-images.githubusercontent.com/4354335/29658663-67a45edc-88c4-11e7-8040-781195bf39e8.png)

![mobile-screen-no-paddings](https://user-images.githubusercontent.com/4354335/29658674-6fd9ca10-88c4-11e7-965a-33109edd73a2.png)